core/src/App.svelte

```diff
@@ -1640,6 +1640,10 @@
       }
 
       if ('luigi.navigation.updateModalDataPath' === e.data.msg) {
+        if(!LuigiConfig.getConfigBooleanValue('routing.showModalPathInUrl')){
+          console.warn('"showModalPathInUrl" will be ignored. Property is not defined or not set to `true`.');
```

Comment:
```suggestion
          console.warn('Updating path of the modal ignored. Property "showModalPathInUrl" is not defined or not set to `true`.');
```
Maybe sth like this is better, since showModalPathInUrl is not the one ignored specifically